Assets/_Scripts/TurnController.cs

```diff
@@ -189,29 +205,82 @@ void Update()
     private void setHeroAction() {
         _heroActionIndex = this.commandSelector.GetChoice();
```

Comment:
We should eventually pull this out from TurnController and create a separate HeroInput object.

Reply:
I'm fine either way, but managing the input is the only unique thing that the TurnController does right now. The rest is done in Battle or BattleFight